docs/cristin.ttl

```diff
@@ -1036,18 +1045,25 @@ It is a self reference.""" ;
     """ .
 
 :antall_sider_totalt
-    a            :cristinfield ;
-    rdfs:comment "The total number of pages. Possibly null." ;
-    rdfs:range   xsd:positiveInteger ;
-    rdfs:domain  :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
-    :isignored   "true" .
+    a                  :cristinfield ;
+    rdfs:comment       "The total number of pages. Possibly null." ;
+    rdfs:range         xsd:positiveInteger ;
+    rdfs:domain        :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :currently_ignored "true" .
 
 :publiseringsar
-    a            :cristinfield ;
-    rdfs:comment "The year of publication for the journal" ;
-    rdfs:range   xsd:positiveInteger ;
-    rdfs:domain  :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
-    :isignored   "true" .
+    a                  :cristinfield ;
+    rdfs:comment       """
+    The year of publication for the Cristin Result. In 99% of the results it is the same as the "arstall".
+    It can be safely ignored.
+     """ ;
+    rdfs:range         xsd:positiveInteger ;
+    rdfs:domain        :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :currently_ignored "true" .
+
+:should_ignore
+         "true"
```

Comment:
Assuming it's intentional that both :currently_ignored and :should_ignore are set.

Formatting a bit wonky.